### PR TITLE
Revert "add role="alert" (#309)"

### PR DESCRIPTION
**Pull request by cvalarida**
This reverts commit 82166c14b685817a38a9fba0663b84afb0e28abc.

## Description
Closes https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/issues/362

## Testing done
None.

## Screenshots
N/A

### CODE REVIEW

**Comment by erikphansen** [comment]
Just a heads up that pulling this into `vets-website` is going to result in test failures. Might want to loop in @kelsonic since he ran into them when the inverse of this PR was pulled into `vets-website`

**Comment by cvalarida** [comment]
Ah, good to know. Thanks, @erikphansen! :bow:

**Comment by kelsonic** [comment]
I think there were only 3 test files I had to update to support `role="alert"` for the `<AlertBox />` in `vets-website`:

https://github.com/department-of-veterans-affairs/vets-website/pull/13607/files#diff-c1b8424ccacefd9a6fe92872f162f55a
https://github.com/department-of-veterans-affairs/vets-website/pull/13607/files#diff-1d2a1deb0c80c4be977bde3281674562
https://github.com/department-of-veterans-affairs/vets-website/pull/13607/files#diff-f810bada69dcd84ed5fe1d35b31a2353

Hope that helps! 🙂

**Comment by erikphansen** [comment]
Ohhhh... looks like those changes have nothing to do with the change to `AlertBox`. So the only tests that will break when `AlertBox` loses the `role="alert"` will be mine. So those tests of mine that will fail in the future can be commented out as a short term fix.

File: packages/formation-react/src/components/AlertBox/AlertBox.jsx [hunk 1]
```diff
@@ -63,7 +63,6 @@ class AlertBox extends React.Component {
         ref={ref => {
           this._ref = ref;
         }}
-        role="alert"
```
Comment (zurbergram) [inline packages/formation-react/src/components/AlertBox/AlertBox.jsx:66]: Could setting `role` be done by passing in a property for the cases where an AlertBox needs a role?

Comment (erikphansen) [inline packages/formation-react/src/components/AlertBox/AlertBox.jsx:66]: We need to bump the version in `package.json` as well.

**Comment by erikphansen** [comment]
Would be super duper if this change got merged and published (along with the corresponding change to `vets-website`) soonish. Anyone using a screen reader has been having a less-than-great experience using VA.gov :(

**Pull request merged**